packages/block-library/src/post-excerpt/edit.js

```diff
@@ -99,21 +100,36 @@ export default function PostExcerptEditor( {
 	const excerptClassName = classnames( 'wp-block-post-excerpt__excerpt', {
 		'is-inline': ! showMoreOnNewLine,
 	} );
+
+	/*
+	 * translators: If your word count is based on single characters (e.g. East Asian characters),
+	 * enter 'characters_excluding_spaces' or 'characters_including_spaces'. Otherwise, enter 'words'.
+	 * Do not translate into your own language.
+	 */
+	const wordCountType = _x( 'words', 'Word count type. Do not translate!' );
+	const currentWordCount = wordCount( rawExcerpt, wordCountType );
+
 	const excerptContent = isEditable ? (
 		<RichText
 			className={ excerptClassName }
 			aria-label={ __( 'Post excerpt text' ) }
 			value={
-				rawExcerpt ||
-				strippedRenderedExcerpt ||
+				rawExcerpt.trim().split( ' ', excerptLength ).join( ' ' ) ||
+				strippedRenderedExcerpt
+					.trim()
+					.split( ' ', excerptLength )
+					.join( ' ' ) ||
```

Comment:
Shouldn't this also take into account the value of `wordCountType`, to ensure that it works correctly for `characters_excluding_spaces` and `characters_including_spaces` too? The implementation here makes sense when the value of `wordCountType` is `words`, but we'll need to accommodate the other 2 cases too

Reply:
Working on this 👍